example-projects/paho-mqtt-example/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "paho-mqtt-example"
+version = "0.2.0"
+authors = ["Subhobrata Dey <sbcd90@gmail.com>"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+async-trait = "^0.1.33"
+cloudevents-sdk = { path = "../sdk-rust" }
+cloudevents-sdk-mqtt = { path = "../sdk-rust/cloudevents-sdk-mqtt"}
+env_logger = "0.7.1"
+paho-mqtt = "0.8"
+serde_json = "^1.0"
+futures = "^0.3"
+tokio = { version = "^0.2", features = ["full"] }
+clap = "2.33.1"
```

Comment:
Here you need to add 

```
[workspace]

```

Like https://github.com/cloudevents/sdk-rust/blob/master/example-projects/warp-example/Cargo.toml